### DISCUSSION TITLE
I am trying to use react-beautiful-dnd with the Table component and gives me this error with the innerRef

### DISCUSSION THREAD

**Discussion by lau-gonzalez** [themesberg/flowbite-react#1377] [Q&A]
the issue is that `ref` is not returning a value.

![image](https://github.com/themesberg/flowbite-react/assets/61595314/4da81980-7d75-4d94-9e8b-c7d5949af6fe)

![image](https://github.com/themesberg/flowbite-react/assets/61595314/5226fa45-d470-4abf-9833-a7acb7132849)

I also tried to console.log the ref value using the useRef hook and the `current` property is `null`.

**Comment by SutuSebastian** [comment]
Known issue, we need to expose refs to the components that do not have it exposed by now. Thanks for reporting!

**Comment by SutuSebastian** [comment]
Fixed by [flowbite-react@0.11.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.11.0) release.
